src/Ogre_glTF.cpp

```diff
@@ -94,7 +97,7 @@ Ogre::MeshPtr loaderAdapter::getMesh() const
 	return Mesh;
 }
 
-Ogre::HlmsDatablock* loaderAdapter::getDatablock() const { return pimpl->materialLoad.getDatablock(); }
+Ogre::HlmsDatablock* loaderAdapter::getDatablock(size_t index) const { return pimpl->materialLoad.getDatablock(index); }
```

Comment:
Same thing. give it a "0" value by default
```suggestion
Ogre::HlmsDatablock* loaderAdapter::getDatablock(size_t index = 0) const { return pimpl->materialLoad.getDatablock(index); }
```

Reply:
This was actually a mistake in this file.